### DISCUSSION TITLE
Automations and Relationships

### DISCUSSION THREAD

**Discussion by Elliot-Construct** [Budibase/budibase#4849] [Help]
Hey Budibase Opensource Community!

I am extremely happy with the general functionality of Budibase and it has the potential to really allow me to interface effectively with my backend and allow me to easily consolidate and manage my data.

However, the in-built automation system is conflicting with Budibase's internaldb relationships.
Binding relationships isn't possible, and in the event you try and update a row that contains relationship data... the update cleans it away thus affecting advanced permissions that I am trying to encode and destroying relationally connected multi-level hierarchies of data in the process.

Is there any way we can fix this... even if it were possible to manually turn off each individual relationship column from updating as a quick bugfix?

I am willing to discuss, engage, and assist with what needs to be done in order to establish cleaner more effective automation. 

Cheers
Elliot

**Comment by mjashanks** [comment]
Thanks for reporting this - definitely a bug - will get this looked at.

**Comment by oswaldB** [comment]
I face the same pb.

When I update a row using automation, it erases all the relations.

**Comment by Elliot-Construct** [comment]
This has been fixed now.

However, I am curious how it might be possible to load multiple relationship options from the automation binding window. I have tried comma-delimited _id bindings but that hasn't worked. [Which is a shame since arrays tend to form comma-delimited]

**Reply by mjashanks** [reply]
Hey @Elliot-Construct  - thanks for reporting this - it's a bug. I have logged it here: https://github.com/Budibase/budibase/issues/4922

In that issue, I've explained how I think it should work! But it does not. Thanks :)
